src/main/java/audiomack/ui/pages/LoginPage.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * @(#) LoginPage.java Copyright (c) 2019 Jala Foundation .
+ * 2643 Av Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of Jala
+ * Foundation, Inc. ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+package audiomack.ui.pages;
+import core.utils.ReaderPropertiesFile;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import audiomack.ui.BasePage;
+
+/**
+ * LoginPage class
+ *
+ * @author Cristian Lujan
+ * @version 0.0.1
+ */
+public class LoginPage extends BasePage {
+
+    @FindBy(id = "react-view")
+    WebElement audiomackForm;
```

Comment:
Shouldn't an access modifier be specified for this attribute(s)?

Reply:
done